frontend/src/components/Page/UnderlinedTitle.jsx

```diff
@@ -5,10 +5,11 @@ import styles from './underlinedTitle.cssmodule.scss';
 
 const UnderlinedTitle = ({ title, big, className, backgroundColor }) => {
   const style = big ? 'bigTitle' : 'title';
+  const titleStyle = big ? '' : 'fr-h2';
 
   return (
     <div className={className}>
-      <h1 className={classNames(styles[style], styles[`${style}-${backgroundColor}`])}>
+      <h1 className={classNames(titleStyle, styles[style], styles[`${style}-${backgroundColor}`])}>
```

Comment:
you could use this syntaxe:
```suggestion
      <h1 className={classNames(styles[style], styles[`${style}-${backgroundColor}`], {'fr-h2': big})}>
```